cloud_experts_tutorials/cloud-experts-deploying-application/cloud-experts-deploying-application-intro.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cloud-experts-deploying-application-intro"]
+= Tutorial: Deploying an application
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: cloud-experts-deploying-application-intro
+
+toc::[]
+
+//rosaworkshop.io content metadata
+//Brought into ROSA product docs 2023-12-14
+
+== Introduction
+
+After successfully provisioning your cluster, you can deploy an application on it. This application will allow us to become more familiar with some of the features of {product-title} and Kubernetes.
```

Comment:
```suggestion
After successfully provisioning your cluster, you can deploy an application on it. This application allows you to become more familiar with some of the features of {product-title} (ROSA) and Kubernetes.
```
I've been changing the first-person plurals when I can to fit more with our normal docs.